src/constants/enums/ingredient-unit.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum IngredientUnit {
```

Comment:
let's move this to unit.enum.ts instead of ingredient-unit.enum.ts.
The reason is simple, unit is not about ingredient only. It is a value object that is universal in our platform